hknweb/candidate/views.py

```diff
@@ -41,21 +42,33 @@ def get_context_data(self):
         challenges = OffChallenge.objects \
                 .filter(requester__exact=self.request.user) \
                 .order_by('-request_date')
-        reviewed_challenges = challenges.filter(reviewed=True)
+        # if either one is waiting, challenge is still being reviewed
+        num_pending = challenges \
+                .filter(Q(officer_confirmed__isnull=True) | Q(csec_confirmed__isnull=True)) \
+                .count()
+        num_rejected = challenges \
+                .filter(Q(officer_confirmed=False) | Q(csec_confirmed=False)) \
+                .count()
+
         announcements = Announcement.objects \
                 .filter(visible=True) \
                 .order_by('-release_date')
+
         today = datetime.date.today()
```

Comment:
Try replacing this with the timezone-aware equivalent:

```
from django.utils import timezone

today = timezone.now()
```

Reply:
I didn't even write this code though

Reply:
While I'm at it I'll just make even more changes